bodo/pandas/physical/join.h

```diff
@@ -92,16 +92,53 @@ class PhysicalJoin : public PhysicalSourceSink, public PhysicalSink {
         std::shared_ptr<bodo::Schema> probe_table_schema_reordered =
             probe_table_schema->Project(probe_col_inds);
 
+        for (const duckdb::JoinCondition& cond : conditions) {
```

Comment:
I don't this needs passing equality conditions and should only be non-equality ones.

Reply:
Yes, I was going to ask you about that.  Made the change.